source/object-model/observers.md

```diff
@@ -30,7 +31,9 @@ var person = Person.create({
   lastName: 'Katz'
 });
 
-person.set('firstName', 'Brohuda'); // observer will fire
+// observer won't fire until `fullName` is consumed first
```

Comment:
what do you think of switching it around to `// consume "fullName" so the observer is triggered`?